### PR TITLE
Lila out streamers online

### PR DESCRIPTION
**Pull request by schlawg**
Allows streamers to show up on broadcast rounds anytime (not just on page load).

### CODE REVIEW

File: src/main/scala/LilaHandler.scala [hunk 1]
```diff
@@ -50,9 +50,9 @@ final class LilaHandler(
       )
     case ApiUserOnline(user, false) => users.tellOne(user, ClientCtrl.ApiDisconnect)
 
-    case Impersonate(user, by) => Impersonations(user, by)
-
-    case Pong(pingAt) => Monitor.ping.record("site", pingAt)
+    case Impersonate(user, by)                                   => Impersonations(user, by)
+    case StreamersOnline(streamers: Iterable[(User.Id, String)]) => Streamer.set(streamers)
```
Comment (ornicar) [inline src/main/scala/LilaHandler.scala:54]: I don't think there's a need or benefit to repeating the `streamers` type here

**Comment by ornicar** [comment]
Is there an associated lila PR coming up?

**Comment by schlawg** [comment]
Yeah, but it's to relay-ui-v2

File: src/main/scala/LilaHandler.scala (hunk 1 above)
Comment (schlawg) [reply src/main/scala/LilaHandler.scala:54]: I agree, let's get rid of some of these types!

**Pull request merged**